README.md

```diff
@@ -66,6 +66,12 @@ Whether to restart the cron daemon after the timezone has changed.
 
 Enable tinker panic, which is useful when running NTP in a VM.
 
+    ntp_tinker_panic: true
```

Comment:
The ordering of these descriptions and examples is off, the `ntp_chrony_maxdistance` should be up here.

Reply:
Can you check now, it shoudl be good, Thanks !